.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Will dependabot be useful, given that we only have Test and Provided dependencies? I'm not sure I want an alert when the test postgres driver makes a patch release for a vuln.